frontend/src/v5/ui/components/viewer/drawingViewer/viewerLayer2D/viewerLayer2D.component.tsx

```diff
@@ -87,7 +85,11 @@ export const ViewerLayer2D = ({ viewBox, active, value, onChange }: ViewerLayer2
 		}
 	}, [active]);
 
-	useEffect(() => { resetArrow(); }, [drawingId]);
+
+	useEffect(() => {
```

Comment:
Something stopped working with the 2d arrow. Now I have to zoom, or pan or click multiple times to get it working.

![arrow-not-working](https://github.com/user-attachments/assets/0deabdc5-faa2-4aa6-b44a-6d17b14efa69)